examples/fluid.py

```diff
@@ -0,0 +1,233 @@
+# http://developer.download.nvidia.com/books/HTML/gpugems/gpugems_ch38.html
+
+import taichi as ti
+import numpy as np
+
+res = 600
+half_res = res * 0.5
+dx = 1.0
+rdx = 1.0 / dx
+half_rdx = 0.5 * rdx
+dt = 0.025
+v_decay = 0.95
```

Comment:
We should probably set this to `1` or `0.999` for now, since the fluid is already too viscous... :-)

Reply:
Done :-)